backend/app/models/connection.py

```diff
@@ -0,0 +1,7 @@
+from pydantic import BaseModel
+
+
+class ConnectionModel(BaseModel):
+    neighbour_name: str
+    from_interface: str
```

Comment:
mozna rozwazyc zrobienie enuma na interfejsy, zeby w stringach był jakiś ład i skład, pod rozkmine dla @OilyCannelloni bo pewnie najwieksze znaczenie bedzie to mialo przy mappingu